data-model/src/main/scala/za/co/absa/enceladus/model/conformanceRule/package.scala

```diff
@@ -80,6 +82,10 @@ package object conformanceRule {
                                     outputColumn: String,
                                     isNullSafe: Boolean = false) extends ConformanceRule {
     override def withUpdatedOrder(newOrder: Int): MappingConformanceRule = copy(order = newOrder)
+
+    override def connectedEntities: Seq[ConnectedEntity] = Seq(
+      ConnectedMappingTable(mappingTable, mappingTableVersion)
```

Comment:
```suggestion
      ConnectedMappingTable(mappingTableName, mappingTableVersion)
```

Reply:
Why this change?

Reply:
Because the field actually signifies the name of the mapping table. `mappingTable` might suggest that it could be an object with subfields or something similar. So I am just suggesting for the field name to change slightly.

The importance of this suggestion is low, so don't be concerned with this too much.

Reply:
While I do 100% agree with you. This is a name of a field which did not change since 1.0.0 and is automatically deserialized. This change would mean we need to update our data model as well and have a migration script for mongo.

Reply:
🤷‍♂️